### PR TITLE
add tag data to EFS resources

### PR DESCRIPTION
**Pull request by skurfer**
The filters shown in #1028 had nothing to work with.

### CODE REVIEW

File: c7n/resources/efs.py [hunk 1]
```diff
@@ -28,6 +28,21 @@ class resource_type(object):
         date = 'CreationTime'
         dimension = None
 
+    def augment(self, filesystems):
```
Comment (kapilt) [inline c7n/resources/efs.py]: this code is extraneous, to just defining a details_spec on the resource_type

Comment (kapilt) [inline c7n/resources/efs.py]: this code is extraneous, to just defining a details_spec on the resource_type

ie. worth trying this one liner, 
```
detail_spec = ("describe_tags", "FileSystemId", "FileSystemId", None) 
```

**Comment by skurfer** [comment]
Some of the other resources were using `augment()` to add tags, so thought that was “the way”.

Your suggested one-line replacement did the trick.

**Comment by kapilt** [comment]
other resources typically do it when they have customization needs beyond the capabilities of details spec (which supports single and multi caller invocations for bulk apis).

**Comment by kapilt** [comment]
not sure why the build isn't passing on travis

**Comment by skurfer** [comment]
Looks like a problem creating a temporary file for the logs tests.

```
E       IOError: [Errno 2] No such file or directory: 'config.json'
```

**Comment by skurfer** [comment]
Rebased against the current master. We’ll see how the tests go this time.

**Pull request merged**